gui/templates/advanced.html

```diff
@@ -208,7 +208,7 @@ <h2>Update Local Settings</h2>
           <input style="text-align:center" id="value" type="number" min="1" max="100" name="value" value="{{ settings.value }}">
           {% elif settings.key == 'AR-Time' %}
             <input style="text-align:center" id="value" type="number" min="1" max="60" name="value" value="{{ settings.value }}">
-          {% elif settings.key == 'AR-MaxFeeRate' or settings.key == 'AF-MaxRate' %}
+          {% elif settings.key == 'AR-MaxFeeRate' %}
```

Comment:
Intentional?

Reply:
The next line has AF-Max and AF-Min fee. Looks like it was redundant here.